CoinDock-app/src/App/Routes/PublicRoutes.js

```diff
@@ -0,0 +1,22 @@
+import { useIsAuthenticated } from "App/Auth/hooks";
+import React from "react";
+import { Route, Routes } from "react-router-dom";
+
+import Login from "Screens/Login/Login";
+import SignUp from "Screens/SignUp/SignUp";
+
+const PublicRoutes = () => {
+  const isAuthenticated = useIsAuthenticated();
+  if (isAuthenticated) {
+    return <React.Fragment />;
+  }
+  return (
+    <Routes>
+      <Route path="/login" element={<Login />} />
+
+      <Route path="/signup" element={<SignUp />} />
```

Comment:
Can you add some default page or redirect to home (dashboard) none of the above routes are matched please